### ISSUE TITLE
Rule jest-dom/prefer-to-have-text-content autofix breaks code

### ISSUE THREAD

**Issue by RIP21** [testing-library/eslint-plugin-jest-dom#51]
- `eslint-plugin-jest-dom` version: 3.0.1
- `node` version: 12.16.1
- `npm` version: 6.13.4

This
```typescript

    expect(container.textContent).toContain(MOCK_REPORT.createdBy)
    expect(container.textContent).toContain(beautifyFullTime(MOCK_REPORT.createdDate))
    expect(container.textContent).toContain(MOCK_REPORT.destinationAuthority)
    expect(container.textContent).toContain(
      beautifyFullTime(MOCK_REPORT.expectedHandlingDate),
    )
    expect(container.textContent).toContain(MOCK_REPORT.externalReference)
    expect(container.textContent).toContain(MOCK_REPORT.internalDecisionRationale)

    markers.forEach(marker => {
      expect(container.textContent).toContain(marker.label)
      expect(container.textContent).toContain(marker.description)
    })

    expect(container.textContent).toContain(humanize(MOCK_REPORT.origin))
    expect(container.textContent).toContain(MOCK_REPORT.reportType)
    expect(container.textContent).toContain(humanize(MOCK_REPORT.state))
    expect(container.textContent).not.toContain(REJECTION_MOCK_INTERACTION.content.reason)
    expect(container.textContent).toContain(beautifyFullTime(MOCK_REPORT.submissionDate))
    expect(container.textContent).toContain(MOCK_REPORT.submittingOfficer)
    expect(container.textContent).toContain(beautifyFullTime(MOCK_REPORT.updatedDate))
    expect(container.textContent).toContain(typologiesAsText)
    expect(container.textContent).toContain(
      (MOCK_SUBJECT.snitch as ReportSubjectFragment).region,
    )
    expect(container.textContent).toContain(MOCK_REPORT.reason)

```
Turns into
```typescript
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)

    markers.forEach(marker => {
      expect(container).toHaveTextContent(/undefined/)
      expect(container).toHaveTextContent(/undefined/)
    })

    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).not.toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
    expect(container).toHaveTextContent(/undefined/)
```

**Comment by benmonro** [comment]
Yikes thanks for reporting. Mind opening a pull request?

**Comment by benmonro** [comment]
fixed in 3.1.4

**Comment by benmonro** [comment]
@all-contributors please add @RIP21 for bugs
